src/anim/anim.js

```diff
@@ -0,0 +1,755 @@
+Object.assign(pc, function () {
+
+    // animation interpolation type
+    var AnimInterpolation = {
+        STEP: 0,
+        LINEAR: 1,
+        CUBIC: 2
```

Comment:
Does this cubic interpolation match what's defined [here](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#appendix-c-spline-interpolation).

Reply:
Correct.